Author: Dejan Latinovic <Dejan.Latinovic@imgtec.com>
Last-Update: Mon, 4 Aug 2014 17:10:38 +0000
Bug-Debian: http://bugs.debian.org/751277
Description: Make sure dnal has a valid return value
 This affects a bug reported against python-biopython.

--- wise-2.4.1.orig/src/models/dnal.c
+++ wise-2.4.1/src/models/dnal.c
@@ -75,6 +75,8 @@
 
   if( show_alb == TRUE )
     dump_ascii_AlnBlock(alb,stdout);
+
+  return 0;
 }
 
 
